#
# lock_acquisition_exception.py
#


class LockAcquisitionException(Exception):
    pass
